backend/src/tasks/ecs-client.ts

```diff
@@ -93,6 +93,9 @@ class ECSClient {
             `DB_NAME=${process.env.DB_NAME}`,
             `DB_USERNAME=${process.env.DB_USERNAME}`,
             `DB_PASSWORD=${process.env.DB_PASSWORD}`,
+            `PE_DB_NAME=${process.env.PE_DB_NAME}`,
```

Comment:
Why add these lines? is it because we need some scans to access PE_DB_USERNAME, etc.?

Reply:
Ya we had this in the red instance. I assumed this was necessary to reach these variables from other "task" scripts